src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -207,13 +207,13 @@ class Request
     protected static $requestFactory;
 
     /**
-     * @param array           $query      The GET parameters
-     * @param array           $request    The POST parameters
-     * @param array           $attributes The request attributes (parameters parsed from the PATH_INFO, ...)
-     * @param array           $cookies    The COOKIE parameters
-     * @param array           $files      The FILES parameters
-     * @param array           $server     The SERVER parameters
-     * @param string|resource $content    The raw body data
+     * @param array                $query      The GET parameters
+     * @param array                $request    The POST parameters
+     * @param array                $attributes The request attributes (parameters parsed from the PATH_INFO, ...)
+     * @param array                $cookies    The COOKIE parameters
+     * @param array                $files      The FILES parameters
+     * @param array                $server     The SERVER parameters
+     * @param null|string|resource $content    The raw body data
```

Comment:
please put null last, it's the less useful type to tell about (personally I don't even get its value at all: it's already on the signature, but anyway...)

Reply:
`array`s are also in the signatures you know ;)